docs/usage/from-a-terminal.rst

```diff
@@ -725,166 +791,274 @@ Global overview
                     [--commit-results-message COMMIT_RESULTS_MESSAGE] [--clean]
```

Comment:
I will generate this section with a script I have locally :-)

Reply:
I'm a bit unsure about what you would like me to do here.....

Reply:
Nothing to do :smile_cat: I just wanted to inform you that that section will be overwritten with the changes made inside the code (what we get from `--help`).